samples/debug/ppi_trace/sysbuild.cmake

```
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(SB_CONFIG_SOC_SERIES MATCHES "nrf52")
```

```suggestion
if(SB_CONFIG_SOC_SERIES_NRF52X)
```

---

For my own understanding, how does this make a difference?

---

Kconfig symbols are checked as part of compliance, if one is changed then it is immediately apparently, checking for strings like this has nothing, it's a complete silent failure until someone reports it as broken